tests/dashboard/test_get_chart_data.py

```diff
@@ -113,11 +117,14 @@ def test_get_data_cols(mock_bucket):
 @mock.patch(
     "src.handlers.dashboard.get_chart_data._build_query",
     lambda query_params, filters, path_params: (
-        "SELECT gender, sum(cnt) as cnt"
-        f'FROM "{TEST_GLUE_DB}"."test_study" '
-        "WHERE COALESCE (race) IS NOT Null AND gender IS NOT Null "
-        "AND gender LIKE 'female' "
-        "GROUP BY gender",
+        (
+            "SELECT gender, sum(cnt) as cnt"
+            f'FROM "{TEST_GLUE_DB}"."test_study" '
+            "WHERE COALESCE (race) IS NOT Null AND gender IS NOT Null "
```

Comment:
There's that odd `Null` again

Reply:
I think this is autocompleteing itself in someplace